cookbooks/aws-parallelcluster-install/recipes/install_mysql_client.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster
+# Recipe:: mysql
+#
+# Copyright:: 2013-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+def install_definition_source_as_repository(repository_name, definition_source, local_file, md5_hash)
```

Comment:
maybe change `install_definition_source_as_repository` to `install_repository_configuration_package`?

Reply:
Renamed.